packages/create-discord-bot/src/index.ts

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env node
+
```

Comment:
Disable eslint rule `n/shebang` for this line

Reply:
I was faster